packages/client/src/constants/leaderBoard.ts

```diff
@@ -0,0 +1,6 @@
+export const leaderboardConstants = {
+  TEAM_NAME: 'FastPaws',
```

Comment:
Странное наименование, название самой константы должен быть заглавными буквами, а параметр наоборот, camelCase-ом
```suggestion
export const LEADERBOARD_CONSTS = {
  teamName: 'FastPaws',
```